.github/scripts/setup_runners.sh

```diff
@@ -68,7 +68,7 @@ fi
 SETUP_ACTION=$1
 SETUP_INSTANCE=`echo $2 | sed 's/,/ /g'`
 SETUP_TOKEN=$3
-SETUP_AMI_ID="ami-0655d7056d3941cd2"
+SETUP_AMI_ID="ami-042f29d2ac35db697"
```

Comment:
Just out of interest, what changed in the AMI?

Reply:
GitHub release a new runner binary out of nowhere and it does not have bc so all runners broke.
Add the python3 installation as well.
If this happens again I would consider making it more dynamic, but they also didnt update broken change in months.